File: 2-Anno/Linguaggi/Java/Pratica/Simulazioni/N40/Noleggio.java
package N40;

public class Noleggio {
	
	private int codiceVeicolo;
	private int numeroGiorni;

	public Noleggio(int codiceVeicolo, int numeroGiorni) {
		this.codiceVeicolo = codiceVeicolo;
		this.numeroGiorni = numeroGiorni;
	}

	public double getCosto() {
		return Main.codiceVeicolo_veicolo.get(codiceVeicolo).getCosto() * numeroGiorni;
	}

	public String toString() {
		return "(" + Main.codiceVeicolo_veicolo.get(codiceVeicolo).getTarga() + ", " + numeroGiorni + ")";
	}
}

File: 2-Anno/Linguaggi/Python/Pratica/Simulazioni/N40/Noleggio.py
class Noleggio:

	def __init__(self, targa, nGiorni, costo) -> None:
		self._targa = targa
		self._nGiorni = nGiorni
		self._costo = costo
	
	def getCosto(self) -> None:
		return self._costo * self._nGiorni
	
	def __str__(self) -> str:
		return "(" + str(self._targa) + "," + str(self._nGiorni) + ")" 